model/src/application/interaction/mod.rs

```diff
@@ -46,7 +44,7 @@ pub enum Interaction {
     /// Application command variant.
     ApplicationCommand(Box<ApplicationCommand>),
     /// Application command autocomplete variant.
-    ApplicationCommandAutocomplete(Box<ApplicationCommandAutocomplete>),
```

Comment:
Should the `ApplicationCommandAutocomplete` variant be removed?

Reply:
Since you have to respond differently to autocomplete interactions, I think we should keep it. So you don't need to match on `ApplicationCommand::kind`.

Reply:
I'll defer to your judgement